src/test/java/nurseybook/logic/commands/AddTagCommandTest.java

```diff
@@ -82,7 +79,7 @@ public void execute_addExistingTagUnfilteredList_success() {
 
     @Test
     public void execute_addTagFilteredList_success() {
-        showElderlyAtIndex(model, INDEX_FIRST);
+        CommandTestUtil.showElderlyAtIndex(model, INDEX_FIRST);
```

Comment:
Similarly, import `CommandTestUtil` at the top of the file would be better?

Recurring in lines 104, 113, 120.